packages/eas-cli/src/credentials/ios/api/AppleTeam.ts

```diff
@@ -2,11 +2,11 @@ import assert from 'assert';
 import gql from 'graphql-tag';
 
 import { graphqlClient } from '../../../api';
-import { Account } from '../../../user/Account';
+import { UserAccount } from '../../../user/Account';
 
 export interface AppleTeam {
```

Comment:
This file should be moved to graphql directory

Reply:
graphql has its own Account interface, which is a superset of this one.